### PR TITLE
Update docs for Databricks 8.2 ML

### PR DESCRIPTION
**Pull request by sameerz**
This is a partial update for the Databricks documentation.  

@viadea can you recommend an init script for users to run 21.06 on 7.3 ML that installs the CUDA 11 toolkit?

### CODE REVIEW

File: docs/get-started/getting-started-databricks.md [hunk 1]
```diff
@@ -11,13 +11,19 @@ At the end of this guide, the reader will be able to run a sample Apache Spark a
...
-    * Azure: 7.3 LTS ML (GPU, Scala 2.12, Spark 3.0.1)
+    * Apache Spark 3.x running in Databricks Runtime 7.3 ML or 8.2 ML with GPU
+    * AWS: 7.3 LTS ML (GPU, Scala 2.12, Spark 3.0.1) or 8.2 ML (GPU, Scala 2.12, Spark 3.1.1)
+    * Azure: 7.3 LTS ML (GPU, Scala 2.12, Spark 3.0.1) or 8.2 ML (GPU, Scala 2.12, Spark 3.1.1)
 
 [Databricks 7.3 LTS
 ML](https://docs.databricks.com/release-notes/runtime/7.3ml.html#system-environment) runs CUDA 10.1
-Update 2, and the initialization scripts will install the appropriate cudf version to match.
+Update 2, and initialization scripts for version 0.5 and earlier will install the appropriate cudf
+version to match.  Users wishing to try 21.06 on Databricks 7.3 LTS ML will need to install the CUDA
```
Comment (tgravescs) [inline docs/get-started/getting-started-databricks.md:21]: do we want to give them example init script that does this?

File: docs/get-started/getting-started-databricks.md [hunk 2]
```diff
@@ -11,13 +11,19 @@ At the end of this guide, the reader will be able to run a sample Apache Spark a
...
+11.0 toolkit on the cluster. 
+
+[Databricks 8.2 ML](https://docs.databricks.com/release-notes/runtime/8.2ml.html#system-environment)
+runs CUDA 11.0, wand initialization scripts for version 21.06 will install the matching cudf
```
Comment (abellina) [inline docs/get-started/getting-started-databricks.md:25]:
```suggestion
runs CUDA 11.0, and initialization scripts for version 21.06 will install the matching cudf
```

**Comment by viadea** [comment]
> This is a partial update for the Databricks documentation.
> 
> @viadea can you recommend an init script for users to run 21.06 on 7.3 ML that installs the CUDA 11 toolkit?

@sameerz I just submitted a PR to add a new init script to install cuda 11.0 with latest 21.06 jars. https://github.com/NVIDIA/spark-rapids/pull/2638

**Review by tgravescs — DISMISSED** [review]

**Comment by viadea** [comment]
@sameerz  Do you want to change the section:
`spark.executorEnv.PYTHONPATH /databricks/jars/rapids-4-spark_2.12-0.4.1.jar:/databricks/spark/python`
to use 21.06 plugin jar?

**Review by viadea — CHANGES_REQUESTED** [review]
`spark.executorEnv.PYTHONPATH /databricks/jars/rapids-4-spark_2.12-0.4.1.jar:/databricks/spark/python` needs to be changed.

**Comment by sameerz** [comment]
> `spark.executorEnv.PYTHONPATH /databricks/jars/rapids-4-spark_2.12-0.4.1.jar:/databricks/spark/python` needs to be changed.

Done

**Review by viadea — APPROVED** [review]

**Comment by sameerz** [comment]
build

**Review by pxLi — APPROVED** [review]

**Pull request merged**